src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -198,9 +198,9 @@ export class InstructorHomePageComponent extends InstructorSessionModalPageCompo
   }
 
   /**
-   * Loads the feedback session in the course.
+   * Loads the feedback session in the course and sorts them according to end date.
    */
-  loadFeedbackSessions(model: CourseTabModel): void {
+  loadFeedbackSessions(model: CourseTabModel, index: number): void {
```

Comment:
You can make this to return void Observable to indicate the completion and subscribe it.

If you think it is overkill, I would suggest just keep parameter `index` as (index and model **refer** to the same thing)